sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -40,11 +44,11 @@ protected RecordedTestBase(bool isAsync, RecordedTestMode mode) : base(isAsync)
             Mode = mode;
         }
 
-        private string GetSessionFilePath(string name = null)
+        private string GetSessionFilePath()
         {
             TestContext.TestAdapter testAdapter = TestContext.CurrentContext.Test;
 
-            name ??= testAdapter.Name;
+            string name = new string(testAdapter.Name.Where(c => !s_invalidChars.Contains(c)).ToArray());
```

Comment:
Do you think it's better to remove them or replace with a placeholder?

Pleceholder option might lead to a fewer conflicts. Take the following example:

"a*"
"*a"

Reply:
I figured the 99% case here would just be a string with double quotes, so just removing them would be the most user friendly, but if there are conflicts in the future it would be hard to fix without re-recording tests. We could remove quotes and replace others, but probably better to just keep it simple. Will replace.